andela/rainier-rc
# #151994137 embed social content

#### What does this PR do?
User can share product on social media that is Facebook, Twitter and Google+
#### How should this be manually tested?
- Run application
- Click on any product 
- You will see the comment box to add reviews and facebook, twitter and Google+ buttons to share product on each social media page
#### Any background context you want to provide?
Uses Disqus comment moderation plugin to give the user a nice commenting experience
#### What are the relevant pivotal tracker stories?
- Story type: Feature
- Story Id: 151994137
- Story: Embed social content 
#### Screenshots (if appropriate)

#### Questions:
None
<img width="1274" alt="screen shot 2017-11-03 at 5 52 20 pm" src="https://user-images.githubusercontent.com/24405172/32391179-6e4ec976-c0d1-11e7-97e3-af2cbd0d25ff.png">
<img width="446" alt="screen shot 2017-11-03 at 5 51 06 pm" src="https://user-images.githubusercontent.com/24405172/32391192-7414e2aa-c0d1-11e7-8235-f8623d08135e.png">
<img width="1283" alt="screen shot 2017-11-03 at 5 56 00 pm" src="https://user-images.githubusercontent.com/24405172/32391194-75b5b1ac-c0d1-11e7-8e76-4b2b7b643dc4.png">
